src/locales/fi.json

```diff
@@ -656,23 +658,27 @@
       "omistajat": {
         "nimi": "$t(form:yhteystiedot:titles:omistaja): $t(form:yhteystiedot:labels:nimi)",
         "email": "$t(form:yhteystiedot:titles:omistaja): $t(form:yhteystiedot:labels:email)",
-        "ytunnus": "$t(form:yhteystiedot:titles:omistaja): $t(form:yhteystiedot:labels:ytunnus)"
+        "ytunnus": "$t(form:yhteystiedot:titles:omistaja): $t(form:yhteystiedot:labels:ytunnus)",
+        "puhelinnumero": "$t(form:yhteystiedot:titles:omistaja): $t(form:yhteystiedot:labels:puhelinnumero)"
```

Comment:
I added there "puhelinnumero" fields because when the phone number is not valid, the error is shown in the missing-field banner.